src/bentoml/_internal/bento/bento.py

```diff
@@ -306,6 +316,40 @@ def from_fs(cls, item_fs: FS) -> Bento:
 
         return res
 
+    def export(
+        self,
+        path: str,
+        output_format: t.Optional[str] = None,
+        *,
+        protocol: t.Optional[str] = None,
+        user: t.Optional[str] = None,
+        passwd: t.Optional[str] = None,
+        params: t.Optional[t.Dict[str, str]] = None,
+        subpath: t.Optional[str] = None,
+    ) -> str:
+        # copy the models to fs
+        models_dir = self._fs.makedir("models", recreate=True)
+        model_store = ModelStore(models_dir)
+        global_model_store = BentoMLContainer.model_store.get()
+        for model in self.info.models:
+            copy_model(
+                model.tag,
+                src_model_store=global_model_store,
+                target_model_store=model_store,
+            )
```

Comment:
if we assume all models are available in local model store (not sure about this when we have model repositories), then maybe we can symlink the models and export should follow symlink. That saves copy time for larger models.

Maybe have another pr to do this so we can merge this pr first?

Reply:
Do you mean adding symlinks into the exported tarball? I think the exported product should be self-contained and can start a bento without cloud access

Reply:
I think the current implementation will copy model to the existing bento's `models/` and then export the bento.

But if you make symlinks to models, the `Exportable`'s export function will still resolve the symlink to the real file and package them inside the bento

Reply:
I understand now, but we needs to consider compatibility on Windows.

Reply:
ye, I think let's do it in another pr.